### ISSUE TITLE
Feature: link wizard to select newsletter page

### ISSUE THREAD

**Issue by webian** [in2code-de/luxletter#52]
Implement the backend link wizard, or something similar, to the field "Where is the newsletter located?" for a more user friendly experience.

**Comment by einpraegsam** [comment]
Yes, I already tried to implement this, but I gave up after some hours of trying. So every PR is welcome here.

**Comment by webian** [comment]
Hi @einpraegsam,
I see... I just started to try this extension and I'm just pinning what I found is missing from direct_mail.

**Comment by rr-it** [comment]
Do it the other way around: Enable pagetree for the luxletter submodule and get the page id from `$_GET['id']`
And then with the help of some new ViewHelpers you can prefill the newsletter title with the page title.

I'll add some proof of concept!
